elasticsearch/Dockerfile.origin

```diff
@@ -0,0 +1,72 @@
+FROM ubi7:latest
```

Comment:
If we intend to remove the centos image what is the advantage of having Dockerfile.origin in lieu of the centos one?

Reply:
The `Dockerfile.centos7` names are just symbolic links to satisfy the trampoline merges with openshift/release/pull/9136. Need to be removed after merging the later. Elasticsearch is the only project we can't use a single Dockerfile due to the ES plugin downloads from brew in the production Dockerfile. I renamed the developer Dockerfile to `Dockerfile.origin` because thi matches the prepare install scripts `prep-install.prod` vs. `prep-install.origin`.

Reply:
I think the naming change make sense if we were to keep it but if the intention is to remove it in future the change is not required and just lends to add noise to the PR

Reply:
FYI, my mistake yesterday, the `Dockerfile.origin` refers to the Dockerfile used by our upstream CI here. AFAIK there is a way without friction for local development and upstream at the same time to access the private koji repos via `fetch-artifacts-koji.yaml`. Therefore we need to keep the two Dockerfile distinct for elastisearch until we figure out how to build elasticsearch incl. all deps and plugins through vendoring on our own. 

In summary:
- `Dockerfile` is the target for our downstream CI used by ART to build images using `fetch-artifacts-koji.yaml`
- `Dockerfile.origin` is the target for our local development and upstream CI and replaces the old centos7 baked image.
- Both files are identical except of the parameters `MAVEN_REPO_URL`, `PROMETHEUS_EXPORTER_VER` and `OPENDISTRO_VER`

I suggest the state as is for now to land the UBI 7 streamlining and re-evaluate a single-Dockerfile strategy for elasticsearch at a later stage.